Source/WebKit/NetworkProcess/storage/NetworkStorageManager.cpp

```diff
@@ -1935,6 +1935,7 @@ void NetworkStorageManager::iterateCursor(const WebCore::IDBRequestData& request
 
 void NetworkStorageManager::getAllDatabaseNamesAndVersions(IPC::Connection& connection, const WebCore::IDBResourceIdentifier& requestIdentifier, const WebCore::ClientOrigin& origin)
 {
+    MESSAGE_CHECK(requestIdentifier.connectionIdentifier().has_value(), connection);
```

Comment:
`MESSAGE_CHECK(requestIdentifier.connectionIdentifier(), connection);`
(std::optional` has `operator bool()` so this should work)